pex/bin/pex.py

```diff
@@ -266,7 +266,7 @@ def configure_clp_pex_environment(parser):
   group.add_option(
       '--interpreter-cache-dir',
       dest='interpreter_cache_dir',
-      default=os.path.expanduser('~/.pex/interpreters'),
+      default=os.path.expanduser('{pex_root}/interpreters'),
```

Comment:
should this also lose the `expanduser` treatment now that we don't specify a string literal with ~ in it?